### PR TITLE
Chronos: Use `ray` output instead of `tqdm` for a clearer log in tuning

### PR DESCRIPTION
**Pull request by liangs6212**

### CODE REVIEW

**Comment by liangs6212** [comment]
Effect Diagram

image

TODO: The "loc" column contains IP and Port information

File: python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py [hunk 1]
```diff
@@ -188,9 +189,10 @@ def run(self):
             search_alg=self._search_alg,
             num_samples=self.num_samples,
             trial_dirname_creator=trial_dirname_creator,
+            callbacks=[CustomProgressCallback()],
```
Comment (TheaperDeng) [inline python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py:192]: I think this Callback is kind of useless, we can remove it for now. Please correct me if it's critical

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

File: python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py (hunk 1 above)
Comment (liangs6212) [reply python/orca/src/bigdl/orca/automl/search/ray_tune/ray_tune_search_engine.py:192]: CustomProgresscallback can avoid unnecessary log output.

**Review by TheaperDeng — APPROVED** [review]

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Chronos-PR-Validation/211/

**Pull request merged**